blocks/library/cover-image/style.scss

```diff
@@ -18,6 +17,10 @@
 		text-align: center;
 	}
 
+	&:not(.components-placeholder) h2 {
```

Comment:
Minor: .components-placeholder is only used in the editor so I think this rule should not be added in style.scss. Maybe we can keep style.scss as it is and a rule in the editor that makes our h2 placeholder the color we want.

Reply:
Totally agree. Amended.